app/classifier/index.cjsx

```diff
@@ -5,6 +5,7 @@ ClassificationSummary = require './classification-summary'
 {Link} = require 'react-router'
 tasks = require './tasks'
 drawingTools = require './drawing-tools'
+{session} = require '../lib/session'
```

Comment:
What doees the { session } do? also will this ever get re required from page navigation, etc?